src/cli/screenshot.rs

```diff
@@ -143,11 +144,14 @@ async fn take_screenshot(
         .http1_ignore_invalid_headers_in_responses(true)
         .trust_dns(true)
         .build()?;
+    let re = Regex::new(r"[<>?.~!@#$%^&*\/|;:']").unwrap();
```

Comment:
I'm not sure, if it's good to use a new crate as a dependency for this only. Do you have any other implementations @enomothem ?

Reply:
@pwnwriter  If you don't use regular, you'll need to use replace() more than once

Reply:
Btw, I'm afraid I don't have a Windows machine to test this pr. Give me some time.